ideas/168-paid-master-nodes/stt-payment-testing.md

```diff
@@ -0,0 +1,47 @@
+# Getting and sending STT, proof of payment
+
+Some notes for initial payment MVP and testing thereof.
+
+## How do I get STT?
+
+- Be on Ropsten
+- Advanced settings -> Development mode
+- Chat with console and use `/faucet <StatusFaucet>` to get ETH
+- > For STT simply send a transaction (can be 0 ether) with Gas limit 105000 (you need to set the value manually!) to the following address: 0x34358C45FbA99ef9b78cB501584E8cBFa6f85Cef (https://wiki.status.im/Testing_FAQ_(core_team))
```

Comment:
Is this address the current default? If not, can we make it one in faucet?

Reply:
That is the contract address for STT, but it's a good point in terms of simplifying flow for testing. Captured issue here https://github.com/status-im/status-react/issues/4062